flash/vision/style_transfer/model.py

```diff
@@ -0,0 +1,90 @@
+from torch import nn
+from torch.nn.functional import interpolate
+
+__all__ = ["Transformer"]
+
+
+class Interpolate(nn.Module):
```

Comment:
Could this model be imported directly from `pystiche`. The goal of Flash is to rely on other libraries to enable tasks, not implement nn layers :)

Reply:
or use the `Resize`/`Rescale` module with those options that you implemented in `kornia` :)